v2/cli/term/project_page.go

```diff
@@ -121,12 +121,11 @@ func (p *projectPage) refresh(projectID string) {
 
 func (p *projectPage) fillProjectInfo(project core.Project) {
 	p.projectInfo.Clear()
+	p.projectInfo.SetTitle(fmt.Sprintf("[yellow]Project: [white]%s\n", project.ID))
```

Comment:
Looking at this in-use, I think it will look cleaner to omit `Project: `. Just use the ID as the title and that's it.